configs/det/dbnet/README.md

```diff
@@ -48,6 +57,17 @@ The overall architecture of DBNet is presented in _Figure 1._ It consists of mul
 
 > More information of DBNet++ is coming soon. The only difference between _DBNet_ and _DBNet++_ is in the _Adaptive Scale Fusion_ module, which is controlled by the `use_asf` parameter in the `neck` module in yaml config file.
 
+### MSRA-TD500
+
+<div align="center">
+
+| **Model**         | **Context**    | **Backbone** | **Pretrained** | **Recall** | **Precision** | **F-score** | **Train T.** | **Throughput** | **Recipe**                  | **Download**                                                                                                                                                                                         |
+|-------------------|----------------|--------------|----------------|------------|---------------|-------------|--------------|----------------|-----------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| DBNet (ours)      | D910x1-MS2.0-G | ResNet-50    | SynthText       | 82.47%     | 87.75%        | 85.03%      | 24.4 s/epoch  | 27.9 img/s      | [yaml](db_r50_td500.yaml) | [ckpt](https://download.mindspore.cn/toolkits/mindocr/dbnet/dbnet_resnet50_td500-0d12b5e8.ckpt)  |
```

Comment:
Why is the Throughput so different from SynthText with the same DBNet?  27.9 vs 82.02 FPS

Reply:
batch_size is different. By adjusting num_workers, the current Throughput for TD500 is 51.1 img/s